AtReconstruction/AtFilter/AtSCACorrect.h

```diff
@@ -23,24 +23,34 @@ class AtSCACorrect : public AtFilter {
    using RawEventPtr = std::unique_ptr<AtRawEvent>;
    AtMapPtr fMap;
 
-   RawEventPtr fBaseline;
-   RawEventPtr fPhase;
+   RawEventPtr fRawEvent;
+
+   Bool_t fDoBaseline;
+   Bool_t fDoPhase;
+   Bool_t fUseChanZero;
```

Comment:
Set these here like:
```
Bool_t fDoBaseline{false};
Bool_t fDoPhase{false};
Bool_t fUseChanZero{false};
```

Should also be `bool` not `Bool_t` since this class is not written to disk